theano/tensor/nnet/neighbours.py

```diff
@@ -208,7 +208,29 @@ def CEIL_INTDIV(a, b):
                                 z_col = j + d * i
 
                                 z[0][z_row, z_col] = ten4[n, s, ten4_2, ten4_3]
-
+                                
+    def infer_shape(self, node, input_shape):
+        in_shape = input_shape[0]
+        def CEIL_INTDIV(a, b):
```

Comment:
This is not suitable for symbolic variables.  Use theano.tensor.ceil_intdiv instead